pkgs/games/katago/default.nix

```
@@ -6,17 +6,20 @@
 , cmake
 , makeWrapper
 , fetchFromGitHub
-, fetchpatch
+, fetchgit
+, writeShellScriptBin
 , cudnn ? null
 , cudatoolkit ? null
 , mesa ? null
 , opencl-headers ? null
 , ocl-icd ? null
+, openssl ? null
```

```suggestion
, openssl
```

Despite being all over the place here this is not required if it is put behind lib.optional(s).

---

These too.